### PR TITLE
Puts digitigrade shoes in Garment Bags

### PR DESCRIPTION
**Pull request by Nobelium-XVIII**
## About The Pull Request
read the tin

## Why It's Good For The Game
less locker bloat = good, probably

shitcode may be present

## Changelog

:cl:
qol: Digitigrade shoes are now in locker Garment Bags.
/:cl:

### CODE REVIEW

**Comment by Nobelium-XVIII** [comment]
wait it actually passed checks

File: fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm [hunk 1]
```diff
@@ -1,4 +1,6 @@
-///Laceups
+/// TODO: Turn into GAGS you lazy fuck ///
```
Comment (JohnFulpWillard) [inline fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm:1]: no

File: fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm [hunk 2]
```diff
@@ -114,3 +116,29 @@
 	worn_icon = 'fulp_modules/features/clothing/digitigrade/icons/digi_feet.dmi'
 	resistance_flags = FIRE_PROOF
 	permeability_coefficient = 0.05
+
+/// Put them into garment bags
+
+/obj/item/storage/bag/garment/captain/PopulateContents()
+	. = ..()
+	new /obj/item/clothing/shoes/brown/digitigrade(src)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm:124]: was this tested?

Comment (Nobelium-XVIII) [reply fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm:124]: Yes and it works

Comment (JohnFulpWillard) [reply fulp_modules/features/clothing/digitigrade/code/digi_shoes.dm:124]: you're mother

**Pull request merged**